static/src/javascripts/projects/commercial/modules/dfp/redplanet.js

```diff
@@ -0,0 +1,49 @@
+// @flow strict
+
+import config from 'lib/config';
+import { commercialFeatures } from "common/modules/commercial/commercial-features";
+import { onIabConsentNotification } from "@guardian/consent-management-platform";
+import { isInAuRegion } from "commercial/modules/header-bidding/utils";
+import { isInVariantSynchronous } from "common/modules/experiments/ab";
+import { commercialRedplanet } from "common/modules/experiments/tests/commercial-redplaner-aus";
```

Comment:
```suggestion
import { commercialRedplanet } from "common/modules/experiments/tests/commercial-redplanet-aus";
```

Will also need the filename changing.